runner/scripts/04-eventing.sh

```diff
@@ -3,39 +3,43 @@
 set -eo pipefail
```

Comment:
I don't think we should test eventing at this point. let's just get serving CI running

Reply:
Removed the file